app/views/ViewUtils.scala

```diff
@@ -24,4 +24,16 @@ object ViewUtils {
   def errorPrefix(form: Form[_])(implicit messages: Messages): String = {
     if (form.hasErrors || form.hasGlobalErrors) messages("error.browser.title.prefix") else ""
   }
+
+  def genericYesNoErrorId = {
+    "value-yes"
+  }
+
+  def percentageExpectedTurnoverErrorId = {
+    "percentageExpectedTurnover.zeroToTwenty"
+  }
+
+  def genericCheckboxErrorId = {
+    "value_0"
+  }
```

Comment:
I know I provided these examples however; I did not give a comprehensive example. Are we sure this covers every page on which the error occurs?

Reply:
It wasn't implemented on the LinkedTransactions page. It is now :)